plugins/scaffolder/src/api.ts

```diff
@@ -164,7 +171,11 @@ export class ScaffolderClient implements ScaffolderApi {
         'Content-Type': 'application/json',
         ...(token && { Authorization: `Bearer ${token}` }),
       },
-      body: JSON.stringify({ templateName, values: { ...values } }),
+      body: JSON.stringify({
+        templateName,
+        values: { ...values },
```

Comment:
i wonder if we could have made the values be JsonValue instead of any, to get rid of the spread

Reply:
To be honest, that type should just be `JsonObject`. But can probably leave that for 1.0